tasktiger/worker.py

```diff
@@ -756,18 +756,21 @@ def _execute_task_group(self, queue, tasks, all_task_ids, queue_lock):
 
         return success, ready_tasks
 
-    def _finish_task_processing(self, queue, task, success):
+    def _finish_task_processing(self, queue, task, success, start_time):
         """
         After a task is executed, this method is called and ensures that
         the task gets properly removed from the ACTIVE queue and, in case of an
         error, retried or marked as failed.
         """
         log = self.log.bind(queue=queue, task_id=task.id)
 
+        when = time.time()
```

Comment:
I'd name it `end_time` or `now`. I think I used `when` sometimes because I'd pass it into `delay(when=when)`, but I don't think it makes any sense in this scenario.